### DISCUSSION TITLE
Enhancement: Cutscene Restart Song Button

### DISCUSSION THREAD

**Discussion by CodeMan98038** [FunkinCrew/Funkin#1671] [Ideas]

**Comment by Windows1Door2Daniel** [comment]
Great IDEA

**Comment by mlamota2020** [comment]
yea like in the tabi mod

**Comment by inue-kang** [comment]
I agree

**Comment by GOOBOZA** [comment]
that will not be a good idea because it will fill up pause menu. what they can add is a skip cut scene button and it skips the cutscenes when you press it. so it does the cutscenes by default when you restart. if you die it will not do that though.
